tests/components/application-connector/README.md

```diff
@@ -229,4 +238,88 @@ To run the tests without removing all the created resources afterwards, run them
 
    ``` shell
    make clean-gateway-test enable-sidecar-after-mtls-test
+   ```
+
+## Application Connectivity Validator
+
+### Design and architecture
+
+The tests consist of:
+- [Test resources](./resources/charts/application-connectivity-validator-test/templates/) used to perform the test
+- [Test runner](./test/application-connectivity-validator/) with all the test cases
+
+The tests are executed as a Kubernetes Job on a Kyma cluster where the tested Application Connectivity Validator is installed. The test Job is deployed in the `test` Namespace.
+![Connectivity Validator tests architecture](./assets/connectivity-validator-tests-architecture.svg)
+
+> **NOTE:** Port `8080` must be excluded from redirection to Envoy, otherwise Connectivity Validator test Pod cannot pass the `X-Forwarded-Client-Cert` header to the Connectivity Validator.
+
+### Building
+
+Pipelines build the Application Connectivity Validator test using the `release` target from the `Makefile`.
+
+To build **and push** the Docker images of the tests, run:
+
+``` sh
+./scripts/local-build.sh {DOCKER_TAG} {DOCKER_PUSH_REPOSITORY}
+```
+
+This will build the following images:
+- `{DOCKER_PUSH_REPOSITORY}/connectivity-validator-test:{DOCKER_TAG}`
+
+### Running
+
+Tests can be run on any Kyma cluster with Application Connectivity Validator.
+
+Pipelines run the tests using the `test-validator` target from the `Makefile`.
+
+#### Deploy a Kyma cluster locally
+
+1. Provision a local Kubernetes cluster with k3d:
+   ```sh
+   kyma provision k3d
+   ```
+
+2. Install the minimal set of components required to run Application Connectivity Validator for Kyma SKR:
+
+    <div>
+    <details>
+    <summary label="SKR">
+    SKR
+    </summary>
+
+    ```bash
+    kyma deploy --components-file ./resources/installation-config/mini-kyma-skr.yaml 
+    ```
+
+    </details>
+    </div>
```

Comment:
``````suggestion
2. Install the minimal set of components required to run Application Connectivity Validator **for Kyma SKR (Compass mode)**:

    ```bash
    kyma deploy --components-file ./resources/installation-config/mini-kyma-skr.yaml 
    ```
``````

We don't need the unfolding panel, as SKR is the only option here.